docs/container.md

```diff
@@ -123,7 +123,7 @@ This feature requires the `apptainer` tool to be installed where the workflow ex
 Nextflow caches those images in the `apptainer` directory in the pipeline work directory by default. However it is suggested to provide a centralised cache directory by using either the `NXF_APPTAINER_CACHEDIR` environment variable or the `apptainer.cacheDir` setting in the Nextflow config file.
 
 :::{versionadded} 21.09.0-edge
-When looking for a Apptainer image file, Nextflow first checks the *library* directory, and if the image file is not found, the *cache* directory is used as usual. The library directory can be defined either using the `NXF_APPTAINER_LIBRARYDIR` environment variable or the `apptainer.libraryDir` configuration setting (the latter overrides the former).
+Nextflow first checks the library directory when retrieving Apptainer images. The cache directory is used if the image file is not found. The library directory can also be defined using the `NXF_APPTAINER_LIBRARYDIR` environment variable. The `apptainer.libraryDir` configuration option overrides `NXF_APPTAINER_LIBRARYDIR` environment variable if both are set.
```

Comment:
I feel this can be improved, it's not clear what's the difference with the cache directory. 

I would do something along the lines:

> Moreover Nextflow uses the library directory to determine the location of the Apptainer container file. Nextflow first checks the *library* directory, if the container file is not found, it check the cache directory. The main difference between the library directory and the cache directory is that the first is assumed to be a read-only container repository, while the latter is expected to be writable path where container files can added for caching purposes. 

> The library directory can be defined either using the `apptainer.libraryDir` configuration setting or the `NXF_APPTAINER_LIBRARYDIR` environment variable. The configuration file option overrides the environment variable if both are set.

Reply:
Thanks, I've worked this so that it explains each setting, and then their relationship. Let me know if it works.